src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -63,50 +59,97 @@ public HttpRequestMessage BuildRequest(FormulaValue[] args)
             // Header names are not case sensitive.
             // From RFC 2616 - "Hypertext Transfer Protocol -- HTTP/1.1", Section 4.2, "Message Headers"
             var headers = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            ByteArrayContent body = null;
 
             var map = _argMapper.ConvertToSwagger(args);
-            foreach (var param in _argMapper._openApiParameters)
+            foreach (var param in _argMapper.OpenApiParameters)
             {
-                if (map.TryGetValue(param.Name, out var paramValue))
+                if (param.In == null && param.Schema.Properties != null && param.Schema.Properties.Any())
+                {
+                    body = GetBody(param, map);
+                }
+                else if (map.TryGetValue(param.Name, out var paramValue))
                 {
                     var valueStr = paramValue.ToObject().ToString();
 
-                    switch (param.In.Value)
+                    if (param.In == null)
                     {
-                        case ParameterLocation.Path:
-                            path = path.Replace("{" + param.Name + "}", HttpUtility.UrlEncode(valueStr));
-                            break;
-
-                        case ParameterLocation.Query:
-                            query.Append((query.Length == 0) ? "?" : "&");
-                            query.Append(param.Name);
-                            query.Append('=');
-                            query.Append(HttpUtility.UrlEncode(valueStr));
-                            break;
-
-                        case ParameterLocation.Header:
-                            headers.Add(param.Name, valueStr);
-                            break;
-
-                        default:
-                            throw new NotImplementedException($"{param.In}");
+                        // No schema, we send the raw value
```

Comment:
>/ No schema, we send the raw value [](http://example.com/codeflow?start=25&length=34)

What's an example that hits this?
I'm wary that we have 2 ways of getting the body (line 69, line 78)

Reply:
[InlineData(3, @"Test.PostWeatherWithId({body: 5})", "POST [http://localhost:5000/weatherPost\r\n](http://localhost:5000/weatherPost\r\n) [content-header] Content-Type: text/json; charset=utf-8\r\n [body] 5")]